thoth/storages/graph/postgres.py

```diff
@@ -4671,6 +4671,80 @@ def sync_dependency_monkey_result(self, document: dict) -> None:
 
                 inspection_run.dependency_monkey_run_id = dependency_monkey_run.id
 
+    def get_python_package_required_symbols(
+        self, package_name: str, package_version: str, index_url: str
+    ) -> List[str]:
+        """Get required symbols for a Python package in a specified version."""
+        package_name = self.normalize_python_package_name(package_name)
+        package_version = self.normalize_python_package_version(package_version)
+
+        with self._session_scope() as session:
+            query = (
+                session.query(PythonPackageVersionEntity)
+                .filter(PythonPackageVersionEntity.package_name == package_name)
+                .filter(PythonPackageVersionEntity.package_version == package_version)
+                .join(
+                    PythonPackageIndex,
+                    PythonPackageIndex.id == PythonPackageVersionEntity.python_package_index_id
+                )
+                .filter(PythonPackageIndex.url == index_url)
+                .join(
+                    HasArtifact,
+                    PythonPackageVersionEntity.id == HasArtifact.python_package_version_entity_id
+                )
+                .join(
+                    RequiresSymbol,
+                    RequiresSymbol.python_artifact_id == HasArtifact.python_artifact_id
+                )
+                .join(
+                    VersionedSymbol,
+                    RequiresSymbol.versioned_symbol_id == VersionedSymbol.id
+                )
+                .with_entities(VersionedSymbol.symbol)
+            )
+
+            # Query returns list of single tuples
+            # NOTE: can be empty even if request is valid
+            to_return = [i[0] for i in query.all()]
+            _LOGGER.debug("The package requires the following symbols: %r", str(to_return))
```

Comment:
Please remove this statement, the string conversion will happen all the time making the debug statement non-optimal.